operational_analysis/methods/electrical_losses.py

```diff
@@ -28,16 +36,35 @@ class ElectricalLosses(object):
     """ 
 
     @logged_method_call
-    def __init__(self, plant, correction_thresh=.95):
+    def __init__(self, plant, UQ = 'Y', num_sim = 20000, uncertainty_meter=0.005, uncertainty_scada=0.005,
+                 uncertainty_correction_thresh=(0.9,0.995), correction_thresh=.95):
         """
         Initialize electrical losses class with input parameters
 
         Args:
          plant(:obj:`PlantData object`): PlantData object from which EYAGapAnalysis should draw data.
-
+         num_sim:(:obj:`int`): number of Monte Carlo simulations
+         UQ:(:obj:`str`): choice whether to perform ('Y') or not ('N') uncertainty quantification
+         uncertainty_meter(:obj:`float`): uncertainty imposed to revenue meter data
+         uncertainty_scada(:obj:`float`): uncertainty imposed to scada data
+         uncertainty_correction_threshold(:obj:`float`): The interval of data availability thresholds (fractions) 
```

Comment:
This is a tuple of floats, instead of a single float. Can you update the data type in the docstring to reflect this?

Reply:
Changed!